### PR TITLE
feat(): add capability to work with outer input

### PR DESCRIPTION
**Pull request by ctrahhuk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,7 +57,8 @@ You can look at different show cases for it here as [Component](https://rawgit.c
            myListFormatter(data: any): string {
               return `(${data[key]}) ${data[name]}`;
             }
-
+  * **`outer-input-element`**, HTMLInputElement reference, path any input from outer scope.
```
Comment (squarepeghires) [inline README.md]: @ctrahhuk did you mean "pass any input from outer scope"

File: README.md [hunk 2]
```diff
@@ -57,7 +57,8 @@ You can look at different show cases for it here as [Component](https://rawgit.c
...
+  * **`extFormControl`**, AbstructControl, path your own FormControl object, can be used with outer-input-element.
```
Comment (squarepeghires) [inline README.md]: @ctrahhuk did you mean "AbstractControl" and again "pass your own FormControl object"

**Comment by ctrahhuk** [comment]
https://squarepeghires.atlassian.net/browse/SP-1393

File: README.md [hunk 3]
```diff
@@ -57,7 +57,8 @@ You can look at different show cases for it here as [Component](https://rawgit.c
            myListFormatter(data: any): string {
               return `(${data[key]}) ${data[name]}`;
             }
-
+  * **`outer-input-element`**, HTMLInputElement reference, pass any input from outer scope.
+  * **`extFormControl`**, AbstructControl, pass your own AbstractControl object, can be used with outer-input-element.
```
Comment (omostepaniuk) [inline README.md]: @ctrahhuk Just a typo crept in here (Abstr**a**ctCont...)

**Review by omostepaniuk — APPROVED** [review]

**Pull request merged**